markdownlint/TOP009_lessonOverviewItemsSentenceStructure/tests/TOP009_invalid_punctuation.md

```diff
@@ -0,0 +1,38 @@
+### Introduction
+
+This file should be marked with an error because one of the lesson overview items is a question.
+
+### Lesson overview
+
+This section contains a general overview of topics that you will learn in this lesson.
+
+- Lesson overview item 1.
+- Lesson overview item 2.
+- Lesson overview item 3?
```

Comment:
I feel like we could combine this and the "no_punctuation" file into one test, since they're essentially testing for the same thing (valid punctuation).